benchmark_runner/workloads/bootstorm_vm.py

```diff
@@ -123,15 +123,15 @@ def _finalize_vm(self):
                 self._data_dict.update(self._prometheus_result)
             total_run_time = self._get_bootstorm_vm_total_run_time()
             self._data_dict.update({'total_run_time': total_run_time})
-        # Google drive run_artifacts_url folder path
-        if self._google_drive_path and self.get_run_artifacts_google_drive():
-            self._data_dict.update({'run_artifacts_url': self.get_run_artifacts_google_drive()})
-        if self._es_host:
-            # upload several run results
-            self._upload_to_elasticsearch(index=self._es_index, kind=self._kind, status=self._status,
-                                          result=self._data_dict)
-            # verify that data upload to elastic search according to unique uuid
-            self._verify_elasticsearch_data_uploaded(index=self._es_index, uuid=self._uuid)
+            # Google drive run_artifacts_url folder path
+            if self._google_drive_path and self.get_run_artifacts_google_drive():
+                self._data_dict.update({'run_artifacts_url': self.get_run_artifacts_google_drive()})
+            if self._es_host:
+                # upload several run results
+                self._upload_to_elasticsearch(index=self._es_index, kind=self._kind, status=self._status,
+                                              result=self._data_dict)
+                # verify that data upload to elastic search according to unique uuid
+                self._verify_elasticsearch_data_uploaded(index=self._es_index, uuid=self._uuid)
```

Comment:
This is so this code is only run if we're not in verification-only mode, correct?

Reply:
Yes, only if it's not in verification-only mode.